pkg/runtime/core/helpers.go

```diff
@@ -46,6 +47,7 @@ func NumberLowerBoundary(input float64) float64 {
 }
 
 func Random(max float64, min float64) float64 {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Can we create a new source instead?

Reply:
Create a new soruce and use this function from stdlib.

Reply:
@ziflex 
Thanks reviewing. your advice makes me confused a little.
First, why `core.Random` cannot be modified directly?
Indeed, it is used for a public function, but it doesn’t call any rand.Seed, thus `rand` has returned same value. Generally, it is wrong way that is getting same values everytime without changing the seed of the pesudo random  except using for testcase which is not allowed different value returns.

Reply:
Sorry, for the confusion. My point was that these functions do pretty much the same, so we could just have 1 implementation, instead of 2.

Second thing that I would like to address is creating a new seed source on each operation to make these function thread safe and give different results every time they are called.

Reply:
After reviewing it one more time with fresh pair of eyes, my comments above can be ignored :)